jxl/src/util/test.rs

```diff
@@ -48,63 +48,71 @@ impl From<JXLError> for Error {
     }
 }
 
-pub fn abs_delta<T: Num + std::cmp::PartialOrd>(left_val: T, right_val: T) -> T {
-    if left_val > right_val {
-        left_val - right_val
-    } else {
-        right_val - left_val
-    }
+pub fn safe_gt<T: std::cmp::PartialOrd>(left_val: T, right_val: T) -> bool {
```

Comment:
Please don't use `safe` for things that have nothing to do with `unsafe` code :-)

This doesn't need to be generic, right? could be done inline in a lambda or so.

Reply:
Fixed